### ISSUE TITLE
MongoDB data loss

### ISSUE THREAD

**Issue by flitzcore** [hagopj13/node-express-boilerplate#289]
I'm using MongoDb Atlas M0 Cluster for this project. It begin when I add a few user and do some testing. I add a few users. but I notice that I'm losing the users. Somehow the MongoDb data got reset. It left me with user collection but no user inside.
Anyone have the same issue? Or anyone know why this happen?

**Comment by waybacktolove** [comment]
yes, this problem is affect to me. i losed mongodb data in all docs

**Comment by waybacktolove** [comment]
have you find the way to resolve this problem?

**Comment by flitzcore** [comment]
unfortunately no. I don't even know what caused it. I can't even replicate the error. All I can do is setup periodic backup

**Comment by waybacktolove** [comment]
Thank you for your response! I am wondering if the data loss might be caused by running MongoDB in Docker. Have you ever tried running MongoDB directly on a server? If so, was it feasible? Also, for your scheduled backup tasks, what would be a good interval for execution?

**Comment by flitzcore** [comment]
I run it in server all this time and still got data loss. I think its because this template use old version of mongodb. I run the backup once every 30 min because my app doesn't have critical data

**Comment by apank** [comment]
This is because you aren't supposed to run tests on your live db; I answered a similar question [here](https://github.com/hagopj13/node-express-boilerplate/issues/294#issuecomment-2272780172)

**Comment by flitzcore** [comment]
Yeah I finally realize that with the hard way....

Pada Rab, 7 Agu 2024 14.16, apank ***@***.***> menulis:
